Formula/sifu.rb

```diff
@@ -0,0 +1,22 @@
+class Sifu < Formula
+  desc "Rapid software development tool made for creating cloud solutions."
```

Comment:
Hi for now it's not open source.

Is this description good enough: 
"DSL based code generation tool for development server-clients applications."